subprojects/files/src/main/java/org/gradle/api/internal/file/AbstractFileCollection.java

```diff
@@ -221,11 +240,21 @@ public boolean contains(File file) {
             public Iterator<File> iterator() {
                 return Iterators.filter(AbstractFileCollection.this.iterator(), predicate);
             }
-        };
-    }
 
-    protected String getCapDisplayName() {
```

Comment:
I guess we can remove the protected method since we already forbid user code to extend `FileCollection`?

Reply:
Let the smoke tests decide this. But I'm pretty sure nobody explicitly references this thing out there. What are the odds that you'd stumble upon it, even if you had this weird use case?